xarray/core/common.py

```diff
@@ -235,7 +239,65 @@ def get_squeeze_dims(xarray_obj, dim, axis=None):
     return dim
 
 
-class BaseDataObject(AttrAccessMixin):
+class SupportsArithmetic(object):  # noqa: W1641
```

Comment:
Why doesn't `# noqa: W1641` disable this warning?

Reply:
Because it's a pylint warning, not a flake8 warning. It's because `py3k` is enabled. We could turn that checker off

https://pylint.readthedocs.io/en/latest/user_guide/message-control.html

Reply:
Yep, that's what I just did.